public/app/plugins/datasource/jaeger/components/SearchForm.tsx

```diff
@@ -144,7 +146,7 @@ export function SearchForm({ datasource, query, onChange }: Props) {
         </InlineField>
       </InlineFieldRow>
       <InlineFieldRow>
-        <InlineField label="Tags" labelWidth={14} grow>
+        <InlineField label="Tags" labelWidth={14} grow tooltip="Values should be in the logfmt format.">
```

Comment:
You may have missed my suggestion here: tooltip should be `Values should be in logfmt.`

Reply:
The reason I wrote it the way I did is to be consistent with the tooltip for the same field in Tempo. So I'd like to confirm whether you'd like me to induce this inconsistency?

Reply:
Done. https://github.com/grafana/grafana/pull/54026/commits/e55a5a64924a182d5a1ff9c4d9419d05b5f36fc5